hack/generate-easyjson.sh

```diff
@@ -14,6 +14,8 @@ cd $TMP_DIR
 
 go mod init tmp-easyjson
 
+echo "replace github.com/go-openapi/strfmt => github.com/kubewarden/strfmt v0.1.2" >> go.mod
```

Comment:
should we backport the generate-easyjson.sh changes to the template?

Reply:
This is not needed by the usual consumers of this policy. This is required here because I had to generate easyjson bindings for Rancher CRDs